src/lib/tls/tls13_pqc/hybrid_public_key.cpp

```diff
@@ -79,13 +97,31 @@ std::vector<size_t> public_value_lengths_for_group(Group_Params group) {
          return {32, 800};
       case Group_Params::HYBRID_X25519_KYBER_768_R3_OQS:
          return {32, 1184};
+      case Group_Params::HYBRID_X25519_eFRODOKEM_640_SHAKE_OQS:
```

Comment:
Not something to address here, but this hardcoding of parameters in something that should theoretically not have to care at all is getting quite dire.